ru_RU.json

```diff
@@ -0,0 +1,956 @@
+  "GAME_CACHE_WAITING_FOR_ONLINE": "Ожидание онлайн, чтобы начать обновление кеша...",
+  "GAME_CACHE_WAITING_FOR_SINGLE_PLAYER": "Ожидание загрузки одиночной игры...",
+  "GENERAL": "Общее",
+  "GET_ALL_WEAPONS": "Получить все оружие",
+  "GIVE_AMMO": "Дать патроны",
+  "GIVE_AMMO_DESC": "Дает игроку боеприпасы",
+  "GIVE_ARMOR": "Дать броню",
+  "GIVE_ARMOR_DESC": "Дает игроку броню",
+  "GIVE_HEALTH": "Дать здоровье",
+  "GIVE_HEALTH_DESC": "Дает игроку немного здоровья",
+  "GIVE_WEAPONS": "Дать оружие",
+  "GIVE_WEAPONS_ALL_DESC": "Дает всем оружие. ",
+  "GIVE_WEAPONS_DESC": "Дает все оружие игроку",
+  "GOD_MODE": "Режим бога",
+  "GODMODE": "Режим бога",
+  "GODMODE_DESC": "Предотвращает получение любого вида урона",
+  "GOLF": "Гольф",
+  "GTA_BANNER": "Баннер GTA",
+  "GUI_TAB_BLACKHOLE": "Черная дыра",
+  "GUI_TAB_CONTEXT_MENU": "Контекстное меню",
+  "GUI_TAB_CREATOR": "Создатель",
+  "GUI_TAB_CUSTOM_TELEPORT": "Пользовательский телепорт",
+  "GUI_TAB_DEBUG": "Отлаживать",
+  "GUI_TAB_ESP": "ESP",
+  "GUI_TAB_ESP_SETTINGS": "ESP",
+  "GUI_TAB_GTA_CACHE_SETTINGS": "Кэш ГТА",
+  "GUI_TAB_GUI": "графический интерфейс",
+  "GUI_TAB_GUI_SETTINGS": "графический интерфейс",
+  "GUI_TAB_HANDLING": "Умение обращаться",
+  "GUI_TAB_HANDLING_CURRENT_PROFILE": "Текущий профиль",
+  "GUI_TAB_HANDLING_SAVED_PROFILE": "Сохраненные профили",
+  "GUI_TAB_HOTKEY_SETTINGS": "Горячие клавиши",
+  "GUI_TAB_IPL": "IPL",
+  "GUI_TAB_LSC": "ЛС кастомс",
+  "GUI_TAB_LUA_SCRIPTS": "Lua-скрипты",
+  "GUI_TAB_MISSIONS": "Миссии",
+  "GUI_TAB_MOBILE": "Мобильный",
+  "GUI_TAB_MODEL_SWAPPER": "Замена модели",
+  "GUI_TAB_NEARBY": "Рядом",
+  "GUI_TAB_NETWORK": "Сеть",
+  "GUI_TAB_ORBITAL_DRONE": "Орбитальный дрон",
+  "GUI_TAB_OUTFIT_EDITOR": "Редактор костюмов",
+  "GUI_TAB_OUTFIT_SLOTS": "Слоты для экипировки",
+  "GUI_TAB_PERSIST_CAR": "Сохранять автомобиль",
+  "GUI_TAB_PERSONAL_VEHICLE": "Личный автомобиль",
+  "GUI_TAB_PLAYER_DATABASE": "База данных игроков",
+  "GUI_TAB_PROTECTION_SETTINGS": "Защита",
+  "GUI_TAB_REACTION_SETTINGS": "Реакции",
+  "GUI_TAB_SELF": "Себя",
+  "GUI_TAB_SESSION": "Сессия",
+  "GUI_TAB_SESSION_BROWSER": "Браузер сеансов",
+  "GUI_TAB_SETTINGS": "Настройки",
+  "GUI_TAB_SPAWN_PED": "Спавн сущности",
+  "GUI_TAB_SPAWN_VEHICLE": "Спавн Транспорта",
+  "GUI_TAB_SPOOFING": "Спуфинг",
+  "GUI_TAB_STAT_EDITOR": "Редактор статистики",
+  "GUI_TAB_SQUAD_SPAWNER": "Создатель отряда",
+  "GUI_TAB_TELEPORT": "Телепорт",
+  "GUI_TAB_TIME_N_WEATHER": "Время и погода",
+  "GUI_TAB_TRAIN": "Поезд",
+  "GUI_TAB_TRANSLATION": "Перевод",
+  "GUI_TAB_VEHICLE": "Транспорт",
+  "GUI_TAB_FUN_VEHICLE": "Забавные функции",
+  "GUI_TAB_WATER": "Вода",
+  "GUI_TAB_WEAPONS": "Оружие",
+  "GUI_TAB_WORLD": "Мир",
+  "GUNSLINGER": "Стрелок",
+  "HANDLING_ACCELERATION_MULTIPLIER": "Множитель ускорения",
+  "HANDLING_ANTI_ROLL_BAR_BIAS_FRONT": "Смещение стабилизатора поперечной устойчивости спереди (0 = спереди, 1 = сзади)",
```

Comment:
What the fuck is this translation? "Transverse stability stabilizer displacement?"